cmd/frontend/graphqlbackend/textsearch.go

```diff
@@ -466,13 +466,15 @@ func zoektSearchHEAD(ctx context.Context, query *search.PatternInfo, repos []*se
 		defer cancel()
 	}
 
-	tr.LogFields(otlog.String("maxWallTime", searchOpts.MaxWallTime.String()))
-
 	t0 := time.Now()
 	resp, err := searcher.Search(ctx, finalQuery, &searchOpts)
-	tr.LogFields(otlog.Int("resp.FileCount", resp.FileCount), otlog.Int("resp.MatchCount", resp.MatchCount), otlog.Object("searchOpts.MaxWallTime", searchOpts.MaxWallTime))
+	tr.LogFields(otlog.Int("resp.FileCount", resp.FileCount), otlog.Int("resp.MatchCount", resp.MatchCount), otlog.Object("searchOpts.MaxWallTime", searchOpts.MaxWallTime), otlog.Object("since(t0)", since(t0)))
```

Comment:
It is nice to split these into multiple lines (other places too)
```suggestion
	tr.LogFields(
		otlog.Int("resp.FileCount", resp.FileCount),
		otlog.Int("resp.MatchCount", resp.MatchCount),
		otlog.Object("searchOpts.MaxWallTime", searchOpts.MaxWallTime),
		otlog.Object("since(t0)", since(t0)),
	)
```